integration/vscode/ada/syntaxes/ada.tmLanguage.json

```diff
@@ -958,6 +1021,10 @@
 				{ "include": "#expression" }
 			]
 		},
+		"multiplying_operator": {
+			"name": "keyword.operator.multiplying.ada",
+			"match": "(?i)\\b(\\*|/|mod|rem)\\b"
```

Comment:
I have this.
![image](https://user-images.githubusercontent.com/34268335/80229683-15891600-8651-11ea-9cae-780c59f9024a.png)

Replace with:
```
"(?i)(\\*|/|\\bmod\\b|\\brem\\b)"
```